op-node/rollup/types.go

```diff
@@ -552,12 +552,24 @@ func (c *Config) IsInteropActivationBlock(l2BlockTime uint64) bool {
 // those timestamps. It can be used for both, L1 and L2 blocks.
 // TODO(12490): Currently only supports Holocene. Will be modularized in a follow-up.
 func (c *Config) IsActivationBlock(oldTime, newTime uint64) ForkName {
-	if c.IsHolocene(newTime) && !c.IsHolocene(oldTime) {
+	if c.IsInterop(newTime) && !c.IsInterop(oldTime) {
+		return Interop
+	} else if c.IsHolocene(newTime) && !c.IsHolocene(oldTime) {
 		return Holocene
 	}
+
 	return ""
 }
 
+func (c *Config) IsActivationBlockForFork(newTime uint64, fork ForkName) bool {
+	if fork == Interop {
+		return c.IsInteropActivationBlock(newTime)
+	} else if fork == Holocene {
+		return c.IsHoloceneActivationBlock(newTime)
+	}
+	return false
+}
+
```

Comment:
Better take the opportunity to add Isthmus and Interop to `rollup.Config.IsActivationBlock` and then reuse it here instead of adding another per-fork switch
```suggestion
func (c *Config) IsActivationBlockForFork(newTime uint64, fork ForkName) bool {
    return fork == c.IsActivationBlock(newTime-c.BlockTime, newTime)
}
```

Reply:
or you could of course also just do that check in the test instead of adding this method to the rollup config

Reply:
Yes I may just call IsInteropActivationBlock() in the test instead, seems simpler.